settings/maven/nexus-settings.xml

```diff
@@ -2,26 +2,40 @@
 <settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">
+  <interactiveMode>false</interactiveMode>
   <mirrors>
     <mirror>
       <id>camunda-nexus</id>
       <mirrorOf>*</mirrorOf>
       <name>Camunda Nexus</name>
-      <url>https://app.camunda.com/nexus/content/groups/public</url>
+      <url>http://repository-ci-camunda-cloud.nexus:8081/content/groups/internal/</url>
```

Comment:
We should not change this file

Reply:
Reverted the changes. Made them during testing.